src/index.js.flow

```diff
@@ -419,6 +419,11 @@ export type DAOConfig<TEntity, TEntityMutator> = {
 };
 
 declare export class DAOResult<TEntity> {
+  constructor(
```

Comment:
I'm not sure if this is the best way to annotate constructor in flow definitions.
I was trying```static <TEntity>(
     count: ?number,
    data: ?TEntity | ?Array<TEntity>,
     error?: Error,
   ): DAOResult<TEntity>;``` like its in immutable.js
but it seems not working